### PR TITLE
update github-script to v6.3.3 resolves #312

### LINKED ISSUE #312: Update to use Environment Files in CI
[receive-pr-and-deploy](https://github.com/ubiquity/ubiquity-dollar/actions/runs/3366042114/jobs/5582129072#step:3:664)
The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

### PR DESCRIPTION
**Pull request by kfukue**
Hi,
I was looking into this issue and it seems like the old version of github-script (3.1.0) was causing the warning message:
"Warning: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/"
The version is a major upgrade so I'm not sure if there are other consequences so please rollback the changes if the workflow fails.

Thanks,
Keita

Resolves #312

https://github.com/ubiquity/ubiquity-dollar/issues/312

### CODE REVIEW

**Review by cuddle-dev — CHANGES_REQUESTED** [review]
Thanks for your first contirbution @kfukue . It looks like your changes still don't resolve the warning.

**Comment by kfukue** [comment]
@0xcodercrane Thanks for your reply! Were you referring to this run when you checked if the change didn't resolve the warning? 
https://github.com/ubiquity/ubiquity-dollar/actions/runs/3501048862/jobs/5864294129

**Comment by cuddle-dev** [comment]
image

Yes, I am. You can look at the warnings in summary menu.

**Comment by kfukue** [comment]
@0xcodercrane I'm not sure if that run included my change since my change is in the actual workflow file... You can see in my screenshot below and the version of github-script is still @3.1.0 (instead of v6.3.3) . I actually replicated this on my personal repository and I saw the warning disappear when I upgraded to the latest version so I'm hoping this will fix this issue.

I'm not sure if there's a different way to this test this (unless you merge this and create a rollback PR just in case) so please let me know what you think!

image

**Comment by cuddle-dev** [comment]
Ok let me merge this and make another pr if it doesn't solve the problem

**Comment by cuddle-dev** [comment]
it caught up another error once I merged it and it also didn't fix version error as well. @kfukue .

**Pull request merged**
